### ISSUE TITLE
Array spread with function call broken in 0.2.0

### ISSUE THREAD

**Issue by AshleyScirra** [babel/minify#695]
#### Input Code

```js
function foo()
{
	return [...bar].reverse();
};
```

#### Actual Output
```js
function foo() {
	return [...bar]; // oops - where did reverse() go?
}
```

#### Expected Output

```js
function foo() {
	return [...bar].reverse();
}
```

#### Details
I'd suspect it is reversing the elements in the array literal and removing the reverse() call, but it does not take in to account spread.
Workaround:
```js
function foo()
{
	const ret = [...bar];
	return ret.reverse();
};
```
This keeps the reverse() call after minify.

**Comment by AshleyScirra** [comment]
Related case: babel-minify crashes with this code:

```js
function foo()
{
	return [...bar].pop();
};
```
> (node:13992) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): TypeError: unknown: Property argument of ReturnStatement expected node to be of a type ["Expression"] but instead got "SpreadElement"

**Comment by boopathi** [comment]
Interesting. I suppose this was fixed in #610.
If you spot the bug, feel free the send a PR.

**Comment by AshleyScirra** [comment]
#628 also looks related btw.
